Sources/Clappr_iOS/Classes/Plugin/Core/Drawer/BottomDrawerPlugin.swift

```diff
@@ -57,15 +57,21 @@ open class BottomDrawerPlugin: DrawerPlugin {
     }
 
     private func moveUp() {
+        enableUserInteractionOnFirstSubview(true)
         view.setVerticalPoint(to: size.height, duration: ClapprAnimationDuration.mediaControlShow)
     }
 
     private func moveDown(with duration: TimeInterval = ClapprAnimationDuration.mediaControlHide) {
+        enableUserInteractionOnFirstSubview(false)
         view.setVerticalPoint(to: hiddenHeight, duration: duration)
     }
 
-    private func addGesture(_ gesture: UIGestureRecognizer) {
-        gesture.cancelsTouchesInView = false
+    private func enableUserInteractionOnFirstSubview(_ enabled: Bool) {
```

Comment:
```suggestion
    private func toggleContentInteraction(enabled: Bool) {
```

I find kinda odd having to pass a boolean to a method called "enable X" in order for it to actually enable something. It's odder still passing `false` to it. Calling it `toggle` and exposing the parameter name kinda solves that, making intention clearer IMHO.

And using "content" instead of "subview" is just to leave it closer to the real objective, instead of how it's actually done.